nucypher/cli/commands/alice.py

```diff
@@ -278,41 +296,34 @@ def create_character(self, emitter, config_file, json_ipc, load_seednodes=True):
     hw_wallet=option_hw_wallet,
     teacher_uri=option_teacher_uri,
     min_stake=option_min_stake,
-    )
+)
 
 
 @click.group()
 def alice():
-    """
-    "Alice the Policy Authority" management commands.
-    """
-    pass
+    """"Alice the Policy Authority" management commands."""
 
 
 @alice.command()
 @group_full_config_options
 @option_config_root
 @group_general_config
 def init(general_config, full_config_options, config_root):
-    """
-    Create a brand new persistent Alice.
-    """
-    emitter = _setup_emitter(general_config)
+    """Create a brand new persistent Alice."""
+    emitter = setup_emitter(general_config)
```

Comment:
We still need to pass banners here, no?
i.e. `setup_emitter(general_config=general_config, banner=ALICE_BANNER)`

If so, this comment applies to other commands that use `setup_emitter` as well, AND the same for other characters as well

Reply:
Same response as below.